cli/server.go

```diff
@@ -1240,6 +1279,15 @@ func (r *RootCmd) Server(newAPI func(context.Context, *coderd.Options) (*coderd.
 	return serverCmd
 }
 
+// templateHelpers builds a set of functions which can be called in templates.
+// We build them here to avoid an import cycle by using coderd.Options in notifications.Manager.
+// We can later use this to inject whitelabel fields when app name / logo URL are overridden.
+func templateHelpers(options *coderd.Options) map[string]any {
+	return map[string]any{
+		"base_url": func() string { return options.AccessURL.String() },
+	}
```

Comment:
Could this just be a `map[string]string`, or do you foresee a need to allow folks to insert values that could change during runtime?

Reply:
See https://pkg.go.dev/text/template#FuncMap

These functions can be parameterized.